sdks/python/apache_beam/examples/snippets/snippets_test_py3.py

```diff
@@ -44,7 +44,7 @@ def test_bad_types_annotations(self):
     # pylint: disable=expression-not-assigned
     # pylint: disable=unused-variable
     class FilterEvensDoFn(beam.DoFn):
-      def process(self, element, *unused_args, **unused_kwargs):
```

Comment:
I put these `unused_` arguments because I ran this file through mypy and it was complaining about them not being there.
PyCharm has similar complaints. Any solution to this?

Reply:
Well, mypy and PyCharm are certainly in error here. Possibly it's because we call DoFns with *args and **kwargs in places, but we construct those to match the actual signature. Please file a JIRA about this.

Reply:
I believe it's because the base class has *args and **kwargs in the prototype:
https://github.com/apache/beam/blob/1de50c348706ed25af2bab9c9477d7d4f36ef8bf/sdks/python/apache_beam/transforms/core.py#L621

Reply:
Hmm... I wonder if it's possible to disable mypy for this method, as it's invoked dynamically and clearly can't figure out what's going on.